va-gov/layouts/home.html

```diff
@@ -29,11 +29,11 @@ <h3><i class="icon-large fa fa-medkit"></i>Health Care</h3>
 
           <h3><i class="icon-large fa fa-handshake"></i>Disability</h3>
           <ul>
-            <li><a href=#>Check your claim or appeal status</a></li>
-            <li><a href=#>View your payment history</a></li>
+            <li><a href="/track-claims">Check your claim or appeal status</a></li>
+            <li><a href="https://www.ebenefits.va.gov/ebenefits/about/feature?feature=payment-history">View your payment history</a></li>
             <li><a href="/disability/upload-supporting-evidence/">Upload evidence to support your claim</a></li>
-            <li><a href=#>Add or remove a dependent</a></li>
-            <li><a href=#>File a claim for compensation</a></li>
+            <li><a href="/disability/add-remove-dependent/">Add or remove a dependent</a></li>
+            <li><a href="/disability/how-to-file-claim/">File a claim for compensation</a></li>
```

Comment:
The latest document has this link removed from the home page: https://docs.google.com/spreadsheets/d/1oTOtVEDBO2FK-0rcNPX6Az6HHFkZrrZrzf-MjG4uEwo/edit#gid=434085583